habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -1098,6 +1116,25 @@ def _eval_checkpoint(
 
                         rgb_frames[i] = []
 
+                    gfx_str = infos[i].get(GfxReplayMeasure.cls_uuid, "")
+                    if gfx_str != "":
+                        write_gfx_replay(
+                            gfx_str,
+                            self.config.TASK_CONFIG.TASK,
+                            current_episodes[i].episode_id,
+                        )
+
+                # episode continues
+                elif len(self.config.VIDEO_OPTION) > 0:
+                    # TODO move normalization / channel changing out of the policy and undo it here
+                    frame = observations_to_image(
+                        {k: v[i] for k, v in batch.items()}, infos[i]
+                    )
+                    if self.config.VIDEO_RENDER_ALL_INFO:
+                        frame = overlay_frame(frame, infos[i])
+
+                    rgb_frames[i].append(frame)
```

Comment:
This also happens line 1084. Are we doubling up all the frames ?

Reply:
Good catch, this was an issue with one of my merges.